src/voice-broadcast/models/VoiceBroadcastRecording.ts

```diff
@@ -27,8 +39,11 @@ interface EventMap {
     [VoiceBroadcastRecordingEvent.StateChanged]: (state: VoiceBroadcastInfoState) => void;
 }
 
-export class VoiceBroadcastRecording extends TypedEventEmitter<VoiceBroadcastRecordingEvent, EventMap> {
+export class VoiceBroadcastRecording
+    extends TypedEventEmitter<VoiceBroadcastRecordingEvent, EventMap>
+    implements IDestroyable {
     private _state: VoiceBroadcastInfoState;
+    private _recorder: VoiceBroadcastRecorder;
```

Comment:
we shouldn't be carrying bad habbits from the TS conversion: please de-underscore the new field.